modules/core/src/main/resources/schemas/neurosciencegraph/core/slicecollection/v0.1.0.json

```diff
@@ -0,0 +1,36 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/core/slicecollection/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/slicecollection/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:SliceCollectionShape",
+      "@type": "sh:NodeShape",
+      "label": "Slice Collection",
+      "comment": "Slice collection.",
+      "targetClass": "nsg:Collection",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/slicecollection/v0.1.0/shapes/SliceCollectionShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:hasPart",
```

Comment:
I would suggest to use dcterms:hasPart.

Reply:
We use nsg:hasPart in other schemas like AnnotatedSlice. Should I still use dcterms:haPart here?

Reply:
Yes. We'll need to update the schemas that use nsg:hasPart too. That can be done in another PR.

Reply:
Should I then also create https://bbp.epfl.ch/nexus/v0/contexts/neurosciencegraph/core/data/v0.4.0 where hasPart is no longer defined as nsg:hasPart but as dcterms:hasPart ?